docs/content/reference/install-configuration/providers/others/http.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "Traefik HTTP Documentation"
+description: "Provide your dynamic configuration via an HTTP(S) endpoint and let Traefik Proxy do the rest. Read the technical documentation."
+---
+
+# Traefik & HTTP
+
+Provide your [install configuration](../overview.md) via an HTTP(S) endpoint and let Traefik do the rest!
+
+## Configuration Example
+
+You can enable the HTTP provider with an endpoint as detailed below:
+
+```yaml tab="File (YAML)"
+providers:
+  http:
+    endpoint: "http://127.0.0.1:9000/api"
+```
+
+```toml tab="File (TOML)"
+[providers.http]
+  endpoint = "http://127.0.0.1:9000/api"
+```
+
+```bash tab="CLI"
+--providers.http.endpoint=http://127.0.0.1:9000/api
+```
+
+## Configuration Options
+
+| Field | Description                                               | Default              | Required |
+|:------|:----------------------------------------------------------|:---------------------|:---------|
+| `providers.providersThrottleDuration` | Minimum amount of time to wait for, after a configuration reload, before taking into account any new configuration refresh event.<br />If multiple events occur within this time, only the most recent one is taken into account, and all others are discarded.<br />**This option cannot be set per provider, but the throttling algorithm applies to each of them independently.** | 2s  | No |
+| `providers.http.endpoint` | Defines the HTTP(S) endpoint to poll. |  N/A    | Yes   |
+| `providers.http.pollInterval` | Defines the polling interval. |  5s    | No   |
+| `providers.http.pollTimeout` | Defines the polling timeout when connecting to the endpoint. |  5s    | No   |
+| `providers.http.headers` | Defines custom headers to be sent to the endpoint. |  5s    | No   |
```

Comment:
The default value here is not 5s, and I think this option should be documented to show how to configure a header to a user, as explained in http://localhost:8000/traefik/providers/http/#headers